benchmark/benchmarks.jl

```diff
@@ -1,10 +1,21 @@
+using BenchmarkTools
 using DelimitedFiles, LinearAlgebra, Printf, SparseArrays
 using Pkg.Artifacts
 
 using DataFrames, MatrixMarket
```

Comment:
It seems you no longer need DataFrames, doesn't it?

Reply:
Yes, you are right. It is no longer needed.